dizoo/gym_anytrading/envs/stocks_env.py

```diff
@@ -0,0 +1,113 @@
+from typing import Any
```

Comment:
correct code format in your modifications like `dizoo/gym_anytrading`, you can refer to [this](https://di-engine-docs.readthedocs.io/zh_CN/latest/21_code_style/index_zh.html)